pkg/lint/lintersdb/enabled_set_test.go

```diff
@@ -117,3 +196,68 @@ func TestGetEnabledLintersSet(t *testing.T) {
 		})
 	}
 }
+
+func TestEnabledSet_combineGoAnalysisLinters(t *testing.T) {
+	m := NewManager(nil, nil)
```

Comment:
proposal: restrict zero value for `NewManager` args and require at least some config (default or empty struct) or some logger

Reply:
the new Manager from the next PR resolve that